app/frontend_messages.py

```diff
@@ -0,0 +1,15 @@
+from flask.ext.babel import gettext as _
+
+
+MESSAGES = {
+            'multiple-surveys': {
+                'TITLE': _("Information"),
+                'MSG': _("Unfortunately you can only complete one survey at a time."),
+                'INSTRUCTIONS': _("Close this window to continue with your current survey."),
+                },
+            }
+
+
+def get_message(message):
```

Comment:
Zen: Explicit is better than implicit: What does this return if message isn't found